src/include/cache_inline.h

```diff
@@ -238,6 +238,28 @@ __wt_session_can_wait(WT_SESSION_IMPL *session)
       FLD_ISSET(session->lock_flags, WT_SESSION_LOCKED_SCHEMA)));
 }
 
+/*
+ * __wt_eviction_clean_pressure --
+ *     Return if an application thread should do eviction due to the total volume of data in cache.
```

Comment:
I would update this comment to be along the lines of:
Return true if the clean cache is stressed and will soon require application threads to evict content.

Reply:
Fixed in [0c400b5](https://github.com/wiredtiger/wiredtiger/pull/10506/commits/0c400b501f4c6d1bd10bdb5d56d7b68d35adfeab)